packages/remark-captions/__tests__/__snapshots__/index.js.snap

```diff
@@ -7,6 +7,8 @@ exports[`#101 1`] = `
 <p><a href=\\"#\\"></a></p>"
 `;
 
+exports[`Two images in figure 1`] = `"<figure><img src=\\"img1.png\\" alt=\\"alt text1\\"><img src=\\"img2.png\\" alt=\\"alt text2\\"><figcaption>Common text</figcaption></figure>"`;
```

Comment:
I am not sure about a small detail: shouldn't every `img` have it's own `figure` tag? I was thinking about something like this, but couldn't find what the best practice was, especially in terms of accessibility:

```html
<figure>
  <figure>
    <img src="img1.png" />
  </figure>
  <figure>
    <img src="img2.png" />
  </figure>
  <figcaption>Global caption</figcaption>
</figure>

Reply:
in fact, figure should not have sub "figure" tag, the full figure is composed of two images and the global caption. So `<figure><img><img><figcaption></figure>` is ok.

As of latex, I will open a question on the template side to see if rendering the "standard way" is ok or if they need a specific command.